build/automation/var/profile/local.mk

```diff
@@ -16,8 +16,8 @@ API_LOG_LEVEL = DEBUG
 DOS_DB_HOST = db-dos
 DOS_DB_NAME = postgres
 DOS_DB_PORT = 5432
-DOS_DB_USERNAME = postgres
-DOS_DB_PASSWORD = postgres
+DOS_DB_USERNAME = uec_dos_api_cs_role
+DOS_DB_PASSWORD = uec_dos_api_cs_role_db_password
```

Comment:
Can we keep `postgres` as a password or alternatively change it to `capacity_status_api`?

Reply:
I think capacity_status_api would be better.

Reply:
good